<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitoring</class>
 <widget class="QWidget" name="Monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>75</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>245</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>415</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>585</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>755</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>925</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>106</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>245</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>415</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>585</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_11">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>755</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_12">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>925</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_begin">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>10</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_end">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>42</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="h_spin_box">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>74</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="Time_check">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>140</y>
     <width>85</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Time_check</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
